src/scenes/OrgRouter/NavBar.tsx

```diff
@@ -256,27 +314,16 @@ export default function NavBar({
           )}
         </SubNavbar>
       </NavItem>
-      {!hasDeliverables && reportsMenu && (
-        <>
-          <NavSection />
-          {reportsMenu}
-        </>
-      )}
-      {hasDeliverables && (
+
+      {(deliverablesMenu || modulesMenu || reportsMenu) && (
```

Comment:
Looks great! I _think_ the reports menu should show up though even if they don't have deliverables or modules. Might want to verify with product, otherwise it looks great

Reply:
I think it will show up if `reportsMenu` is not null! And that is decided by `reports.length` and `canSubmitReports`. The behavior should be unchanged.

Reply:
Oh I see! `acceleratorSectionTitle` is just an empty string without modules or deliverables, thanks!